docs/proposals/images/machinepool-machines/inframachinepool-create-replicas.plantuml

```diff
@@ -0,0 +1,17 @@
+@startuml inframachinepool-create-replicas
+:InfraMachinePool create replicas reconciliation;
```

Comment:
I'm not sure what this means, the InfraMachinePool is an object, it shouldn't "create" anything

in the diagram it looks like an action / first step

Reply:
I mean to say that this is the flow for the InfraMachinePool controller when we need to scale up. Although a reconcile loop checks for both delete and create, I wanted to indicate that it's split it into two separate diagrams like in AzureMachinePool PR comments.